include/internal/quic_ssl.h

```diff
@@ -127,6 +127,15 @@ QUIC_CHANNEL *ossl_quic_conn_get_channel(SSL *s);
 int ossl_quic_has_pending(const SSL *s);
 int ossl_quic_get_shutdown(const SSL *s);
 
+/*
+ * Set QLOG diagnostic title. String is copied internally on success and need
+ * not remain allocated. Only has any effect if logging has not already begun.
+ * For use by tests only. Setting this on a context affects any QCSO created
+ * after this is called but does not affect QCSOs already created from a
+ * context.
+ */
+int ossl_quic_set_diag_title(SSL_CTX *ctx, const char *title);
+
```

Comment:
I'm sure I'm missing something, but I don't see where this title gets used.  I would have expected it to get embedded somewhere in the json stream, but I don't

Reply:
It is definitely used, see output for example:

```
{"qlog_version":"0.3","qlog_format":"JSON-SEQ","title":"quic_multistream_test: script 52","trace":{"common_fields":{"time_format":"delta","protocol_type":["QUIC"],"system_info":{"process_id":315810}},"vantage_point":{"type":"client"}}}
```

It stashes the title in the `SSL_CTX` which is then used by `port_make_channel` in `wuic_port.c`. `ossl_quic_channel_new` then copies it when instantiating QLOG.